packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1962,11 +1962,6 @@ export default class Meeting extends StatelessWebexPlugin {
       });
     }
 
-    // Should PIN collection be followed by pin propt ???
-    if (options.hostPin) {
-      Metrics.pinCollection({meeting: this});
-    }
```

Comment:
@chrisadubois  thoughts on this . we can check moderator flag and then send metrics but do we need to send !?

Reply:
@arun3528 I removed this line because `Metrics.pinCollection` doesn't exist. Surprised there wasn't more errors around it.

We are still sending the metrics on pin collection here: https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/@webex/plugin-meetings/src/meeting/util.js#L193

Reply:
thanks @adamweeks